kratos/processes/calculate_discontinuous_distance_to_skin_process.h

```diff
@@ -66,11 +67,18 @@ class KRATOS_API(KRATOS_CORE) CalculateDiscontinuousDistanceToSkinProcess : publ
         ModelPart& rVolumePart,
         ModelPart& rSkinPart);
 
+    /// Constructor with option
+    CalculateDiscontinuousDistanceToSkinProcess(
+        ModelPart& rVolumePart,
+        ModelPart& rSkinPart,
+        const Flags rOptionEdge);
+
     /// Constructor with options
     CalculateDiscontinuousDistanceToSkinProcess(
         ModelPart& rVolumePart,
         ModelPart& rSkinPart,
-        const Flags rOptions);
+        const Flags rOptionEdge,
+        const Flags rOptionExtraEdge);
```

Comment:
Now I understand why you created the new argument...

Options must be a unique `Flag` argument. Note that flags can be concatenaded meaning that the same flag variable contains at the same time `CALCULATE_ELEMENTAL_EDGE_DISTANCES` and `CALCULATE_ELEMENTAL_EXTRAPOLATED_EDGE_DISTANCES`.

Reply:
Makes sense. How do I concatenate it though and how do I get it "out" again separately? Is there an example somewhere? Can I extract the information like this: `if (mOptions.Is(CalculateDiscontinuousDistanceToSkinProcessFlags::CALCULATE_ELEMENTAL_EDGE_DISTANCES))` ?

Reply:
you can concatenate them by doing `myFlag = FLAG_1 || FLAG_2`. Then if you do `myFlag.Is(FLAG_1)` it will return `true`. In the same way, `myFlag.Is(FLAG_2)` will return `true` too.

Reply:
Thanks!:)